components/pack/include/PackFile.h

```diff
@@ -43,6 +43,91 @@ struct G3DTextureHeader
 
 static_assert(sizeof(G3DTextureHeader) == 4 * sizeof(uint32_t));
 
+struct SadAudioMetaDataHeader
+{
+	char unknown[27];
+	uint32_t entries;
+};
+
+struct SadAudioBankSampleTable
+{
+	uint16_t numOfEntries;
+	uint16_t numOfEntries2;
```

Comment:
Is this value ever used?

Are we sure numOfEntries isn't just a single 32 bit int or maybe numOfEntries2 is just padding?

Reply:
It's not padding. Both fields are the same value for some reason.